xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -198,8 +198,16 @@ The examined parameter should be greater than or equal to the examined position
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Cancels to currently pending or if none is pending next call to <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" />, without completing the <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Cancels the pending <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation without causing it to throw annd without completing the <see cref="T:System.IO.Pipelines.PipeReader" />. If there is no pending operation, this cancels the next operation.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The canceled <xref:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)> operation returns a <xref:System.IO.Pipelines.ReadResult> where <xref:System.IO.Pipelines.ReadResult.IsCanceled> is true.
```

Comment:
How do I verify that I didn't mess up the syntax or references locally before submitting a PR?

Reply:
- Go to this PR's CI section.
- Expand the "Show all checks" link.
- Look at the OpenPublishing.Build leg - It should say "Validation status: passed". Not "warnings" or "errors".
- Click on the "Details" link.
- In the new window, scroll down to the "Validated files" table.
- For any of the files you modified in your PR, which will show up in the table, click on the "View" link under the "Preview URL" column.
- You'll be able to see your changes as they would look in MS Docs (this is just a preview site).

For this particular API, [this is the preview page](https://review.docs.microsoft.com/en-us/dotnet/api/system.io.pipelines.pipereader.cancelpendingread?view=dotnet-plat-ext-2.1&branch=pr-en-us-5033).